src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
 'use strict';
 
-// write your code here
+const tbody = document.querySelector('tbody');
+const scoreField = document.querySelector('.game-score');
+
+const btnReset = document.createElement('button');
+
+const messageStart = document.querySelector('.message-start');
+const messageLose = document.querySelector('.message-lose');
+const messageWin = document.querySelector('.message-win');
+
+btnReset.classList.add('button');
+btnReset.classList.add('restart');
+btnReset.textContent = ('Restart');
+
+const rows = tbody.rows.length;
+const cells = rows;
+
+let board;
+let tempBoard;
+
+let score = 0;
+
+const matrix = [...tbody.children].map(row => [...row.children]);
```

Comment:
Consider using more descriptive variable names instead of `matrix`. For example, `gameBoard`. 🤔